src/reducers/ByProgramReducer.js

```diff
@@ -35,7 +35,7 @@ const actions = {
 
 const STEPS = {
   requisition: {
-    program: ['program', 'supplier', 'orderType', 'period'],
+    program: ['supplier', 'program', 'orderType', 'period'],
```

Comment:
Reordering to supplier->program as part of program supplier requisition refactoring (Maybe shouldv'e been in a different PR.. oops)